### ISSUE TITLE
[FR]Add option to remember the lsat used label type

### ISSUE THREAD

**Issue by martonmiklos** [inventree/InvenTree#4932]
### Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find a similar feature request

### Problem statement

When printing labels the label type is always set to the first label type available. 
When printing labels with the label type other than the first the label type always need to be selected.
This is could be quite inconvenient.

### Suggested solution

Set autocomplete flag on the label type comboboxes.

If approved I can fix it quite easily.

### Describe alternatives you've considered

Adding field/table for the a "favourite" label type per user per label type (part/stock/stocklocation).

### Examples of other systems

N/A

### Do you want to develop this?

- [X] I want to develop this.

**Comment by SchrodingersGat** [comment]
That's quite reasonable, this could easily be a user setting (or one user setting per type of label).

**Comment by martonmiklos** [comment]
> That's quite reasonable, this could easily be a user setting (or one user setting per type of label).

Do we want to provide UI for this purpose or we just want to remember the last used (i.e. extend the autocomplete behavior profile -not browser session- wide)?

**Comment by SchrodingersGat** [comment]
I don't think it needs to be exposed to the settings UI, but could easily be done as a future step if we wanted.
